deployment/src/main/java/io/quarkiverse/quinoa/deployment/framework/FrameworkType.java

```diff
@@ -6,11 +6,14 @@
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.AbstractMap.SimpleImmutableEntry;
 import java.util.Arrays;
 import java.util.Locale;
+import java.util.Map;
 import java.util.Objects;
 import java.util.Optional;
 import java.util.Set;
+import java.util.StringJoiner;
```

Comment:
I just learned something new. I have been using Commons Lang `StringUtils.join` for years didn't know it was in the JDK now!